beta/src/pages/learn/render-and-commit.md

```diff
@@ -1,44 +1,44 @@
+* क्यों रेंडरिंग हमेशा एक डोम अद्यतन उत्पन्न नहीं करता है
 
 </YouWillLearn>
 
-Imagine that your components are cooks in the kitchen, assembling tasty dishes from ingredients. In this scenario, React is the waiter who puts in requests from customers and brings them their orders. This process of requesting and serving UI has three steps:
+कल्पना कीजिए कि आपके कंपोनेंट्स रसोई में रसोइया हैं, सामग्री से स्वादिष्ट व्यंजन इकट्ठा करते हैं। इस परिदृश्य में, React वेटर है जो ग्राहकों से अनुरोध करता है और उन्हें उनके आदेश लाता है। UI का अनुरोध करने और उसे प्रस्तुत करने की इस प्रक्रिया के तीन चरण हैं
 
-1. **Triggering** a render (delivering the guest's order to the kitchen)
-2. **Rendering** the component (getting the order from the kitchen)
-3. **Committing** to the DOM (placing the order on the table)
+1. **ट्रिग्गरिंग** रेंडर (रसोईघर में अतिथि का आदेश पहुंचाना)
+2. **रेंडरिंग** कॉम्पोनेन्ट (रसोई से आदेश प्राप्त करना)
+3. **कमिटिंग** डोम पर (टेबल पर ऑर्डर देकर)
 
 <IllustrationBlock sequential>
-  <Illustration caption="Trigger" alt="React as a server in a restaurant, fetching orders from the users and delivering them to the Component Kitchen." src="/images/docs/illustrations/i_render-and-commit1.png" />
-  <Illustration caption="Render" alt="The Card Chef gives React a fresh Card component." src="/images/docs/illustrations/i_render-and-commit2.png" />
-  <Illustration caption="Commit" alt="React delivers the Card to the user at their table." src="/images/docs/illustrations/i_render-and-commit3.png" />
+  <Illustration caption="ट्रिगर" alt="React as a server in a restaurant, fetching orders from the users and delivering them to the Component Kitchen." src="/images/docs/illustrations/i_render-and-commit1.png" />
+  <Illustration caption="रेंडर" alt="The Card Chef gives React a fresh Card component." src="/images/docs/illustrations/i_render-and-commit2.png" />
+  <Illustration caption="कमिट" alt="React delivers the Card to the user at their table." src="/images/docs/illustrations/i_render-and-commit3.png" />
 </IllustrationBlock>
 
-## Step 1: Trigger a render {/*step-1-trigger-a-render*/}
+## स्टेप 1: एक रेंडर ट्रिगर करें {/*step-1-trigger-a-render*/}
 
-There are two reasons for a component to render:
+किसी कॉम्पोनेन्ट को रेंडर करने के दो कारण हैं:
 
-1. It's the component's **initial render.**
-2. The component's **state has been updated.**
+1. यह कॉम्पोनेन्ट का **प्रारंभिक रेंडर है।**
+2. कॉम्पोनेन्ट की **स्थिति अपडेट कर दी गई है।**
 
-### Initial render {/*initial-render*/}
+### प्रारंभिक रेंडर {/*initial-render*/}
 
-When your app starts, you need to trigger the initial render. Frameworks and sandboxes sometimes hide this code, but it's done by calling `ReactDOM.render` with your root component and the target DOM node:
+जब आपका ऐप शुरू होता है, तो आपको शुरुआती रेंडर को ट्रिगर करना होगा। फ़्रेमवर्क और सैंडबॉक्स कभी-कभी इस कोड को छुपाते हैं, लेकिन यह आपके मूल कॉम्पोनेन्ट और टारगेट DOM नोड के साथ ReactDOM.render को कॉल करके किया जाता है:
```

Comment:
`मूल कॉम्पोनेन्ट` => `रुट कौम्पोनॅन्ट`

Reply:
Hey, @arshadkazmi42 one more question should performance be written as it is in Hindi or translate it to the Hindi meaning of it?

Reply:
Lets write it in hindi, without translation, it will make more sense